tests/TestCase/ORM/TableTest.php

```diff
@@ -26,9 +26,9 @@
 use Cake\Event\Event;
 use Cake\Event\EventManager;
 use Cake\I18n\Time;
-use Cake\ORM\AssociationCollection;
 use Cake\ORM\Association\BelongsToMany;
 use Cake\ORM\Association\HasMany;
+use Cake\ORM\AssociationCollection;
```

Comment:
@markstory this is wrong. ORM\AssocationCollection comes *after* ORM\Association\HasMany